napari/utils/colormaps/colormap.py

```diff
@@ -44,6 +46,7 @@ class Colormap(EventedModel):
     # fields
     colors: Array[float, (-1, 4)]
     name: str = 'custom'
+    display_name: str = 'custom'
```

Comment:
I have a slight concern here that people creating their own colormaps are going to be be confused about why they need both a `name` and a `display_name`, and the we could get a lot of colormaps which have a `name` specified, but still have `'custom'` as their `display_name`. This came up recently here https://github.com/napari/napari/pull/2483#discussion_r605155630 too and the proposal was ultimately to use `display_name` too, so maybe its ok and this just becomes something standard. Thoughts @tlambert03 @jni?

Reply:
How about we have `display_name=None` and if not provided we take `display_name=name`. This will let API users who are just adding a temporary colormap within a session not have to worry about it, but will allow us to translate all our default ones. Thoughts?

Reply:
I like this!

Reply:
Fixing